Source/Core/DolphinWX/Debugger/DSPRegisterView.h

```diff
@@ -44,4 +44,7 @@ class DSPRegisterView : public wxGrid
 public:
 	DSPRegisterView(wxWindow* parent, wxWindowID id = wxID_ANY);
 	void Update() override;
+
+private:
+	CDSPRegTable* m_register_table;
```

Comment:
Added comments. I should have done that initially, since the construction of them doesn't take in a parent argument, which might lead to confusion.